File: src/main/java/com/freya02/botcommands/pagination/transformer/IMentionableTransformer.java
package com.freya02.botcommands.pagination.transformer;

import net.dv8tion.jda.api.entities.IMentionable;

public class IMentionableTransformer implements EntryTransformer<IMentionable> {
	@Override
	public String toString(IMentionable mentionable) {
		return mentionable.getAsMention();
	}
}

File: BotCommands-core/src/main/kotlin/io/github/freya022/botcommands/api/pagination/menu/transformer/IMentionableTransformer.kt
package io.github.freya022.botcommands.api.pagination.menu.transformer

import net.dv8tion.jda.api.entities.IMentionable

class IMentionableTransformer : EntryTransformer<IMentionable> {
    override fun toString(entry: IMentionable): String = entry.asMention
}
